src/System.Threading.Timer/tests/TimerFiringTests.cs

```diff
@@ -156,7 +154,7 @@ public void Running_Timer_CanBeFinalizedAndStopsFiring()
     {
         AutoResetEvent are = new AutoResetEvent(false);
         TimerCallback tc = new TimerCallback((object o) => are.Set());
-        var t = new Timer(tc, null, 100, 500);
+        var t = new Timer(tc, null, 10, 500);
```

Comment:
Can make this 1 instead of 10, though as noted above I expect they'll actually behave the same.